psiturk/psiturk_shell.py

```diff
@@ -988,10 +985,12 @@ def hit_create(self, numWorkers, reward, duration):
         if duration <= 0:
             print '*** duration must be greater than 0'
             return
-        
 
+        if not self._confirm_dialog('Create HIT? [y/n]: '):
```

Comment:
I was envisioning some way to nicely display the options that will be used for the HIT if the user says 'y', perhaps putting [this code](https://github.com/jacob-lee/psiTurk/blob/8be8f6b8481e3e839afacee84ad333853d8fdd43/psiturk/psiturk_shell.py#L1049) into a function and repeating that, with some wording change to say something along the lines of "about to create a hit with the following settings"

Reply:
sure, I'll work on it.